scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -87,15 +86,8 @@ cd $BUILD_DIR
 cmake \
   -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/emscripten.cmake \
   -DCMAKE_BUILD_TYPE=Release \
-  -DBoost_FOUND=1 \
   -DBoost_USE_STATIC_LIBS=1 \
   -DBoost_USE_STATIC_RUNTIME=1 \
-  -DBoost_INCLUDE_DIR="$WORKSPACE"/boost_1_68_0/ \
-  -DBoost_FILESYSTEM_LIBRARY_RELEASE="$WORKSPACE"/boost_1_68_0/libboost_filesystem.a \
```

Comment:
This is the reason for updating the boost version for emscripten and reworking the build scripts a bit: This kind of overwrite won't be possible anymore (at least without hoops) when moving to modern imported targets.